examples/chat_app.md

```diff
@@ -0,0 +1,251 @@
+# Building a chat app with Deno
+
+## Introduction
+
+In this tutorial we'll create a simple chat app using Deno. Our chat app will
+allow multiple chat clients connected to the same backend to send group messages
+through web sockets. After a client chooses a username, they can then start
+sending group messages to other online clients. Each client also displays the
+list of currently active users.
+
+![Untitled](https://s3-us-west-2.amazonaws.com/secure.notion-static.com/4002626d-edc6-41dd-a518-c6197df4f450/Untitled.png)
```

Comment:
this image doesn't seem to display as well as doesn't have appropriate alt text to explain it